tls/s2n_cipher_preferences.c

```diff
@@ -30,8 +30,13 @@
  * TLS1.3 compatibility. */
 #define S2N_TLS13_CIPHER_SUITES_20190801 \
     &s2n_tls13_aes_256_gcm_sha384,       \
-        &s2n_tls13_aes_128_gcm_sha256,   \
-        &s2n_tls13_chacha20_poly1305_sha256
+    &s2n_tls13_aes_128_gcm_sha256,       \
```

Comment:
Should `aes_128` here also be first?

Reply:
I'll leave this to team to decide - changing this right now would create a cascading changes to existing tests